packages/docs-reanimated/docs/layout-animations/keyframe-animations.mdx

```diff
@@ -96,117 +25,68 @@ const keyframe = new Keyframe({
     transform: [{ rotate: '45deg' }],
   },
 });
+
+function App() {
+  return <Animated.View entering={keyframe} />;
+}
 ```
```

Comment:
I'd add `Type definitions` section under the reference. Users need it to see a complete signature of the function/class etc. You can try looking at, for example, one in [useScrollViewOffset docs page](http://localhost:3000/react-native-reanimated/docs/scroll/useScrollViewOffset)

Reply:
🆗 ✅